ldms/src/core/ldms_core.h

```diff
@@ -193,15 +195,31 @@ struct ldms_set_hdr {
 	uint8_t pad3;	/* data pad */
 	uint32_t card;		/* Size of dictionary */
 	uint32_t meta_sz;	/* size of meta data in bytes */
-	uint32_t data_sz;	/* size of metric values in bytes */
+	uint32_t data_heap_sz;	/* size of metric values in bytes */
```

Comment:
If __data_sz__ includes both the static + heap data size, then I think __data_sz__ is the correct name. Otherwise, I think that __data_heap_sz__ is confusing.